api/api_sources/schema-files/watercraftRiskAssessment.schema.yaml

```diff
@@ -15,31 +16,140 @@ schemas:
     displayLayout: {}
     layout: {}
     computedFields: {}
+    # Relations
+    relations: 
+      journeys:
+        header:
+          key: 'watercraftAssessment.journeys'
+          default: Water Craft Journeys
+        description: 
+          key: 'watercraftAssessment.journeys.description'
+          default: 'Associated Water-bodies'
+        type: array
+        relationshipType: one-to-many
+        schema: WatercraftJourneySchema
+        meta: 
+          embedded: true 
+    # End: Relations
     columns: 
+      # Primary Column
       id: 
         name: 'watercraft_risk_assessment_id'
         comment: 'Auto generated primary key'
         definition: SERIAL PRIMARY KEY
+      # Time
       timestamp: 
-        name: timestamp
+        name: 'timestamp'
         comment: 'Date and time of watercraft observation.'
         definition: 'TIMESTAMP NOT NULL'
-      highRiskAssessmentForm:
-        name: high_risk_assessment_form
-        comment: 'Stored JSON structure for whole high risk assessment form data'
-        definition: JSONB NULL
-      lowRiskAssessmentForm:
-        name: low_risk_assessment_form
-        comment: 'Stored JSON structure for whole low risk assessment form data'
-        definition: JSONB NULL
-      fullObservationForm:
-        name: full_observation_form
-        comment: 'Stored JSON structure for full observation questionnaire'
-        definition: JSONB NULL
-      additionalInfo:
-        name: additional_info
-        comment: 'Stored JSON structure for additional information'
-        definition: JSONB NULL
+      # Boolean
+      passportHolder:
+        name: 'pass_port_holder_ind'
```

Comment:
should be 'passport_holder_ind' - 'passport' is more intuitive than 'pass_port'

Reply:
Resolved